### ISSUE TITLE
Custom GIF image in Letters loader

### ISSUE THREAD

**Issue by vishwaskale51** [Gix075/jqueryIntroLoader#9]
How can i add custom GIF loader in Letter loader animation. Plz reply on this mail vishwaskale51@gmail.com its urgent.

**Comment by Gix075** [comment]
This feature is not included in jqueryIntroLoader plugin, so there aren't an option for add a custom gif in letter animation. But you can try to load the gif as a background of the loader element using css.
So if you have the following markup
```
<div id="element" class="introLoading"></div>
```
you can style the **.introLoading** class adding to this the gif image as a background-image.
```
.introLoading { 
    background-image: url(your-image.gif);
    background-repeat: no-repeat;
    background-position: center center; /* use this rule to place the loader above or under the animated letters. */
}
```
I hope this can help you.

**Comment by vishwaskale51** [comment]
Hi,

       *Thanks for reply so fast. Grateful to you.*


Thanks & Regards
Vishwas Kale


*Website <http://vishwaskale.com/> | Blog <http://vishwaskale.com/blogs> |
LinkedIn <https://in.linkedin.com/in/vishwaskale>+91 9890156145*

**Comment by vishwaskale51** [comment]
Below CSS working fine you have sent but GIF still display when "Page Ready!" text displaying i want to hide GIF when "Page Ready!" text displaying. See Screenshot. Please help

![1](https://user-images.githubusercontent.com/29252334/26913762-9ba0c76e-4c3a-11e7-9493-b28fe26b71df.png)



.introLoading { 
    background-image: url(your-image.gif);
    background-repeat: no-repeat;
    background-position: center center; /* use this rule to place the loader above or under the animated letters. */
}

**Comment by Gix075** [comment]
No, I'm sorry it's impossible. You can only use the spin js before the lettersLoader animation.

**Comment by vishwaskale51** [comment]
Hi,

*          Ok no problem thanks for help.*


Thanks & Regards
Vishwas Kale


*Website <http://vishwaskale.com/> | Blog <http://vishwaskale.com/blogs> |
LinkedIn <https://in.linkedin.com/in/vishwaskale>+91 9890156145*

**Comment by Gix075** [comment]
Thanks to you for using my plugin!
